packages/sui-svg/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@s-ui/svg",
+  "version": "1.0.0",
+  "description": "Converts svg files into React Components",
+  "main": "index.js",
+  "scripts": {
+    "test": "exit 0"
+  },
+  "bin": {
+    "sui-svg": "./bin/sui-svg.js"
+  },
+  "dependencies": {
+    "@s-ui/component-peer-dependencies": "latest",
+    "@compositor/kit": "1.0.47",
+    "@s-ui/bundler": "3",
+    "@s-ui/helpers": "1",
+    "@s-ui/js": "1",
+    "@svgr/core": "3.1.0",
+    "commander": "2.19.0",
+    "styled-components": "4.0.2",
```

Comment:
???

Reply:
It is a dependency of Kit

Reply:
A peer dependency? :/ We might want to re-check this in the future and extract the behaviour that compositor is giving to us as this doesn't seem to be reasonable.